api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentStatusController.java

```diff
@@ -145,6 +146,19 @@ public ResponseEntity<String> paymentStatusSecond(@PathVariable("failureReferenc
         return new ResponseEntity<>(SUCCESSFUL_OPERATION, HttpStatus.OK);
     }
 
+    @ApiOperation(value = "update payment reference for unprocessed payment", notes = "update payment reference for unprocessed payment")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "payment reference updated successfully")
+    })
+    @PatchMapping(value = "/jobs/unprocessed-payment-update")
+    @Transactional
```

Comment:
can this annotation be moved to service layer method please?

Reply:
moved to service